sdk/appcenter/src/androidTest/java/com/microsoft/appcenter/AppCenterAndroidTest.java

```diff
@@ -23,15 +23,25 @@
 import java.util.concurrent.atomic.AtomicReference;
 
 import static org.junit.Assert.*;
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Matchers.isNull;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
 
 @SuppressWarnings("unused")
 public class AppCenterAndroidTest {
 
+    private static final String DUMMY_APP_SECRET = "123e4567-e89b-12d3-a456-426655440000";
+
+    private static final String DUMMY_TRANSMISSION_TARGET_TOKEN = "snfbse234jknf";
+
     private Application mApplication;
 
     @Before
     public void setUp() throws IllegalAccessException, ClassNotFoundException, InstantiationException {
         AppCenter.unsetInstance();
+        DummyService.sharedInstance = null;
```

Comment:
unsetInstance is the pattern we use.

Reply:
Not for dummy ones in tests.
https://github.com/Microsoft/AppCenter-SDK-Android/blob/b4d77a54661fb55a5dd5ec997ab50818a6f0480d/sdk/appcenter/src/test/java/com/microsoft/appcenter/AppCenterTest.java#L130-L131

Do you want to change it everywhere?

Reply:
Ah dummy ones, ok. Somehow I thought it was on the new transmission classes. Dummy is fine.